Syntax error in shutdown script for py-youwol --daemonize

Use [shellcheck](https://www.shellcheck.net/), which can be integrated in IDE.

Shell script is not POSIX sh (see [SC3010](https://github.com/koalaman/shellcheck/wiki/SC3010)):
```
/home/runner/work/_temp/py-youwol-integration/py-youwol.shutdown.sh: 6: [[: not found
/home/runner/work/_temp/py-youwol-integration/py-youwol.shutdown.sh: 12: [[: not found
```
Additionally the shebang is missing its bang (i.e. `!`) (see [SC1113](https://github.com/koalaman/shellcheck/wiki/SC1113)):
```
#/bin/sh
```

--- a/youwol/utils/utils_low_level.py
+++ b/youwol/utils/utils_low_level.py
@@ -209,18 +209,18 @@ async def execute_shell_cmd(cmd: str, context: Context, log_outputs=True):
 
 
 def shutdown_daemon_script(pid: int) -> str:
-    return f"""#/bin/sh
+    return f"""#!/bin/sh
 py_youwol_pid={pid}
 ## Sanity check
 program_name=$(ps -p $py_youwol_pid -o command=)
 echo "$program_name" | grep -q py-youwol
-if [[ $? -ne 0 ]]; then
+if [ $? -ne 0 ]; then
     echo "Pid $py_youwol_pid does not look like py-youwol - program name is '$program_name'
 Aborting"
     exit
 fi
 kill $py_youwol_pid
-if [[ $? -eq 0 ]]; then
+if [ $? -eq 0 ]; then
     echo "Successfully send kill signal"
 else
     echo "Failed to send kill signal"

